tests/httpwrappers/tests.py

```diff
@@ -530,6 +530,34 @@ def test_content_type_mutually_exclusive(self):
                 headers={"Content-Type": "text/csv"},
             )
 
+    def test_text_property(self):
+        response = HttpResponse("Hello, world!")
+        self.assertEqual(response.text, "Hello, world!", "Initial content test failed.")
+
+        response.content = "Updated content"
+        self.assertEqual(
+            response.text, "Updated content", "Content update test failed."
+        )
+
+        response = HttpResponse(
```

Comment:
Suggestion: This might benefit from a `self.subTest` to parameterise the test

Reply:
Thank you so much for your guidance. The code looks much cleaner and more organized now. :)